plugins/by-name/blink-cmp/default.nix

```
@@ -42,4 +42,18 @@ lib.nixvim.plugins.mkNeovimPlugin {
     };
     signature.enabled = true;
   };
+
+  extraOptions = {
+    setupLspCapabilities = lib.nixvim.options.mkEnabledOption "setup LSP capabilities for blink-cmp";
+  };
+
+  extraConfig = cfg: {
+    plugins.lsp.capabilities =
```

```suggestion
   # TODO: On Neovim 0.11+ and Blink.cmp 0.10+ with vim.lsp.config, you may skip this step. This is still required when using nvim-lspconfig until [this issue is completed](https://github.com/neovim/nvim-lspconfig/issues/3494)
    plugins.lsp.capabilities =
```

Should probably add a todo for this since it will become obsolete